bigbluebutton-html5/imports/ui/components/audio/audio-modal/container.jsx

```diff
@@ -22,7 +23,11 @@ export default lockContextContainer(withModalMounter(withTracker(({ mountModal,
   const listenOnlyMode = getFromUserSettings('bbb_listen_only_mode', APP_CONFIG.listenOnlyMode);
   const forceListenOnly = getFromUserSettings('bbb_force_listen_only', APP_CONFIG.forceListenOnly);
   const skipCheck = getFromUserSettings('bbb_skip_check_audio', APP_CONFIG.skipCheck);
+  const skipCheckOnJoin = getFromUserSettings('bbb_skip_check_audio_on_first_join', APP_CONFIG.skipCheckOnJoin);
+  const autoJoin = getFromUserSettings('bbb_auto_join_audio', APP_CONFIG.autoJoin);
   const meeting = Meetings.findOne({ meetingId: Auth.meetingID }, { fields: { voiceProp: 1 } });
+  const getEchoTest = Storage.getItem('getEchoTest', false);
```

Comment:
`Storage.getItem()` does not have a second parameter, so it will return `null` instead of `false` if the item is not found

Reply:
Updated in https://github.com/bigbluebutton/bigbluebutton/pull/11284/commits/620dd8db830dc2708cebf9668d86c27b77b8498c